pydm/widgets/timeplot_curve_editor.py

```diff
@@ -0,0 +1,71 @@
+from ..PyQt.QtGui import QDialog, QVBoxLayout, QHBoxLayout, QTableView, QAbstractItemView, QSpacerItem, QSizePolicy, QDialogButtonBox, QPushButton, QItemSelection
+from ..PyQt.QtCore import Qt, pyqtSlot
```

Comment:
@laispc which version of PyQT are you using? 4.8? 4.4? 5?
For PyQt4... QItemSelection is part of QtGui: http://pyqt.sourceforge.net/Docs/PyQt4/qitemselection.html

And for PyQt5 it is part of QtCore...
https://doc.qt.io/qt-5/qitemselection.html

@mattgibbs ... what about add a variable on the PyQt module inside PyDM to define the version of PyQT being used and do this check?